server/src/Ogmios/Query.hs

```diff
@@ -0,0 +1,102 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+
+module Ogmios.Query (
+  makeRequest,
+  tryQueryUntilZero,
+  defaultServerParameters,
+  ServerParameters (..),
+) where
+
+import Control.Exception.Base (SomeException, try)
+import Data.Aeson qualified as Aeson
+import Data.Bifunctor (first)
+import Data.ByteString.Lazy (ByteString)
+import Data.Text (Text)
+import GHC.Generics
```

Comment:
We always use explicit imports, unless it's something like `Prelude` in Purescript (which always needs to be imported)